Description: Do not require neither wget nor curl to build
Author: Reinhard Tartler <siretart@debian.org>
Last-Update: 2014-09-27

--- a/make/configure.py
+++ b/make/configure.py
@@ -1018,6 +1018,7 @@
         self.name     = name
         self.pool     = pool
         self.kwargs   = kwargs
+        self.selected = None
 
     def _action( self ):
         self.session = []
@@ -1476,7 +1477,7 @@
         xcodebuild = ToolProbe( 'XCODEBUILD.exe', 'xcodebuild', abort=False )
         lipo       = ToolProbe( 'LIPO.exe',       'lipo', abort=False )
 
-        fetch = SelectTool( 'FETCH.select', 'fetch', ['wget',wget], ['curl',curl] )
+        fetch = SelectTool( 'FETCH.select', 'fetch', ['wget',wget], ['curl',curl], abort=False ) 
 
     ## run tool probes
     for tool in ToolProbe.tools:
